crates/fsrt/src/test.rs

```diff
@@ -1230,6 +1230,154 @@ fn graphql_compass() {
     dbg!(scan_result.into_vulns()[0].description());
 }
 
+#[test]
```

Comment:
Can you add cases where the global and export uses `process.env` to make sure we don't file results for those?

Reply:
Fixed. Thank you!